jme3-android-examples/jme3-android-tests/app/build.gradle

```diff
@@ -0,0 +1,53 @@
+plugins {
+    id 'com.android.application'
+}
+
+android {
+    compileSdkVersion 30
+    buildToolsVersion "30.0.2"
+
+    defaultConfig {
+        applicationId "org.jmonkeyengine.jme3androidexamples"
+        minSdkVersion 22
+        targetSdkVersion 30
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    configurations.all {
+        exclude group:"org.jmonkeyengine",module:"jme3-desktop"
+        exclude group: 'jbullet', module:'jbullet'
+        exclude group: 'stack-alloc', module:'stack-alloc'
+    }
+}
+
+dependencies {
+
+    implementation 'androidx.appcompat:appcompat:1.2.0'
+    implementation 'com.google.android.material:material:1.3.0'
+    implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
+    testImplementation 'junit:junit:4.13.2'
+    androidTestImplementation 'androidx.test.ext:junit:1.1.2'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
+    /*add the JMonkeyEngine Dependencies*/
+    runtimeOnly project(":stack-alloc")
+    implementation project(":jbullet")
+    implementation "org.jmonkeyengine:jme3-core:3.4.0-alpha7"
+    implementation "org.jmonkeyengine:jme3-examples:3.4.0-alpha7"
+    implementation "org.jmonkeyengine:jme3-jbullet:3.4.0-alpha7"
+    implementation "org.jmonkeyengine:jme3-effects:3.4.0-alpha7"
+    implementation "org.jmonkeyengine:jme3-android-native:3.4.0-alpha7"
+    implementation "org.jmonkeyengine:jme3-lwjgl:3.4.0-alpha7"
```

Comment:
why `jme3-lwjgl` is in dependencies?

Reply:
@Ali-RS thanks for reviewing,
yeh, the author who made this android example, uses all jmonkeyengine examples through direct gradle dependiceis (of test-data & examples) & classNames, i haven't dig deeper inside jme3-examples, so i thought may be at least one example would use something lwjgl related, but you are right lwjgl is useless for android.